templates/types/extractor/fastapi/app/ui/components/monaco.py

```diff
@@ -0,0 +1,61 @@
+"""Reflex custom component Monaco."""
+
+# For wrapping react guide, visit https://reflex.dev/docs/wrapping-react/overview/
+# Taken and modified from https://github.com/Lendemor/reflex-monaco
+
+import reflex as rx
+
+
+class MonacoComponent(rx.Component):
+    """Base Monaco component."""
+
+    library = "@monaco-editor/react@4.6.0"
```

Comment:
Great! is this working well? I also had a try on this: https://github.com/react-simple-code-editor/react-simple-code-editor
But getting error with Reflex on NextJS config and Typescript issue.

Reply:
@leehuwuj yes this worked great. I love it